src/audit/auth-resources/spring/transform.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns:context="http://www.springframework.org/schema/context"
+       xmlns:util="http://www.springframework.org/schema/util"
+       xsi:schemaLocation="
+    http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd
+    http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context.xsd
+    http://www.springframework.org/schema/util http://www.springframework.org/schema/util/spring-util.xsd">
+
+
+  <context:property-placeholder/>
+
+  <!-- fcrepo-transform settings -->
+  <bean id="cacheDir" class="java.io.File">
+    <constructor-arg type="java.lang.String" value="${fcrepo.home:/tmp}/ldcache"/>
```

Comment:
We may run into trouble and list question when the default `/tmp` does not work on all systems.